source/MaterialXFormat/Util.cpp

```diff
@@ -172,6 +172,7 @@ void flattenFilenames(DocumentPtr doc, const FileSearchPath& searchPath, StringR
                 for (size_t i = 0; i < searchPath.size(); i++)
                 {
                     FilePath testPath = searchPath[i] / resolvedValue;
+                    testPath.normalize();
```

Comment:
Wouldn't we need to call `testPath = testPath.getNormalized()` to actually affect the result?

Reply:
Typo fixed.